traitsui/editor.py

```diff
@@ -175,18 +168,34 @@ def prepare(self, parent):
         self.update_editor()
 
     def init(self, parent):
-        """ Finishes initializing the editor by creating the underlying toolkit
-            widget.
+        """ Creating the underlying toolkit for the widget.
+
+        This method must be overriden by subclasses.  Implementations must
+        ensure that the :attr:`control` trait is set to an appropriate
+        toolkit object.
+
+        Parameters
+        ----------
+        parent : toolkit control
+            The parent toolkit object of the editor's toolkit objects.
         """
         raise NotImplementedError
 
     def set_focus(self):
         """ Assigns focus to the editor's underlying toolkit widget.
+
+        This method must be overriden by subclasses.
```

Comment:
It might make sense to put this text into the `NotImplementedError` exception message.